test/creditguild-test.js

```diff
@@ -0,0 +1,47 @@
+const { expect } = require("chai");
+const { ethers } = require("hardhat");
+
+describe("CreditGuild", function () {
+    
+    let membershipFee = 20;
+    let vouchAmout = 1;
+    let marketRegistry = "0x4ABAb00eae341cF2E79D2ae35455749F10B39dEC";
+    let unionToken = "0x16e3271d3A8F0d0ef4F9d06De9f805eB34885b39";
+    let underlyingToken = "0x453814a3f8e5F4bBE40948B7F22fd658Edd76101";
+
```

Comment:
For both unionToken and underlyingToken you can deploy this contract:
https://github.com/unioncredit/union-v1-contracts/blob/master/contracts/tests/FaucetERC20.sol